modules/ROOT/pages/config-mutual-auth.adoc

```diff
@@ -12,10 +12,13 @@ Before enabling mutual authentication, verify the following endpoint and port ar
 * EU control plane:
 ** `configuration-resolver.prod-eu.msap.io:443`
 
+[IMPORTANT]
+Failure to meet the connectivity requirements above before enabling Mutual Authentication will result in errors while creating/managing application deployments in the Runtime Fabric.
```

Comment:
Just a few minor edits: "Failure to meet the previously described connectivity requirements before enabling mutual authentication results in errors with creating/managing application deployments in Runtime Fabric."